fastlane/Fastfile

```diff
@@ -37,7 +37,7 @@ platform :android do
             begin
                 upload_sample_app(APPCENTER_ADVANCED_SAMPLE_APP_NAME)
             rescue => exception
-                UI.error(exception)
+                UI.user_error!(exception)
```

Comment:
I'm actually not so sure about these specific changes. Should these be `user_error` or remain as `error`. What's the difference in output?